pkg/port/models.go

```diff
@@ -193,7 +193,8 @@ type EntityMappings struct {
 }
 
 type Port struct {
-	Entity EntityMappings `json:"entity"`
+	Entity       EntityMappings `json:"entity"`
+	ItemsToParse string         `json:"ItemsToParse"`
```

Comment:
it should be under `EntityMapping` as u can have multiple mappings for the same kind in the k8s exporter and you might want different parsing for each

Reply:
I need the if so I can check wether the items array parse is neede, but I need fix it to use only one loop as you suggest with the nice idea of insert the single object into array.

Reply:
I had put it under entity and not entityMappings because it is make more sense when you define the entity items to parse and the mapping separately.
The mappings will look like this now:
```yaml
- kind: apps/v1/deployments
    selector:
      Query: .item.name | startswith("itamar")
    port:
      entity:
        itemsToParse: .spec.template.spec.containers
        mappings:
          - identifier: .item.name + "-Container"
            title: .item.name
            blueprint: '"container"'
            icon: '"Deployment"'
            properties:
              image: .item.image
            relations:
              deployment: .metadata.name + "-Deployment-" + .metadata.namespace
```
What do you think?

Reply:
on other places we place the items to prase in the same level of the blueprint
Can u ask bogu what we'd prefer? on the entity level or the mapping level?